app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    if command[0] == 'cp' and command[1] != command[2]:
```

Comment:
We recommend using double quotes everywhere
Single quotes should be used if double quotes are used inside the f-string
so, please. use here double quotes